howto/regex.po

```diff
@@ -32,160 +32,250 @@ msgstr ""
 #: ../Doc/howto/regex.rst:18
 msgid ""
 "This document is an introductory tutorial to using regular expressions in "
-"Python with the :mod:`re` module.  It provides a gentler introduction than "
-"the corresponding section in the Library Reference."
+"Python with the :mod:`re` module.  It provides a gentler introduction than the "
+"corresponding section in the Library Reference."
 msgstr ""
+"Ce document est un tutoriel d'introduction à l'utilisation des expressions "
+"régulières en Python avec le module :mod:`re`. Il fournit une introduction plus "
+"abordable que la section correspondante dans la Bibliothèque de Référence"
 
 #: ../Doc/howto/regex.rst:24
 msgid "Introduction"
 msgstr "Introduction"
 
 #: ../Doc/howto/regex.rst:26
 msgid ""
-"Regular expressions (called REs, or regexes, or regex patterns) are "
-"essentially a tiny, highly specialized programming language embedded inside "
-"Python and made available through the :mod:`re` module. Using this little "
-"language, you specify the rules for the set of possible strings that you "
-"want to match; this set might contain English sentences, or e-mail "
-"addresses, or TeX commands, or anything you like.  You can then ask "
-"questions such as \"Does this string match the pattern?\", or \"Is there a "
-"match for the pattern anywhere in this string?\".  You can also use REs to "
-"modify a string or to split it apart in various ways."
-msgstr ""
+"Regular expressions (called REs, or regexes, or regex patterns) are essentially "
+"a tiny, highly specialized programming language embedded inside Python and made "
+"available through the :mod:`re` module. Using this little language, you specify "
+"the rules for the set of possible strings that you want to match; this set "
+"might contain English sentences, or e-mail addresses, or TeX commands, or "
+"anything you like.  You can then ask questions such as \"Does this string match "
+"the pattern?\", or \"Is there a match for the pattern anywhere in this string?"
+"\".  You can also use REs to modify a string or to split it apart in various "
+"ways."
+msgstr ""
+"Les expressions régulières (appelées REs, ou regexes, ou motifs regex) sont "
```

Comment:
Please also quote "expressions rationnelles", widely used by non-users of "expression régulières". I would also split "motifs" and "regex", I *never* heard of "motifs regex", yet I heard of "regex" and wikipedia heard of "motifs".